### PR TITLE
dragonball: support boot_info.initrd

### PR DESCRIPTION
**Pull request by ZhangShuaiyi**
BootSourceConfig has already support initrd_path, pass boot_info.initrd to BootSourceConfig.initrd_path.

Fixes: #6915

### CODE REVIEW

File: src/libs/kata-types/src/config/hypervisor/dragonball.rs [hunk 1]
```diff
@@ -122,14 +122,14 @@ impl ConfigPlugin for DragonballConfig {
...
+            if db.boot_info.image.is_empty() && db.boot_info.initrd.is_empty() {
                 return Err(eother!(
-                    "Guest boot image for dragonball hypervisor is empty"
+                    "Both guest boot image and initrd for dragonball hypervisor are empty"
                 ));
             }
-            if !db.boot_info.initrd.is_empty() {
-                return Err(eother!("Initrd for dragonball hypervisor should be empty"));
-            }
+            // if !db.boot_info.initrd.is_empty() {
```
Comment (Ji-Xinyou) [inline src/libs/kata-types/src/config/hypervisor/dragonball.rs:130]: Remove the segment if you dont want this code.

**Review by Ji-Xinyou — CHANGES_REQUESTED** [review]
One minor problem, others lgtm.

**Comment by Ji-Xinyou** [comment]
Also, you need a sign-off for commit message.

**Comment by ZhangShuaiyi** [comment]
@Ji-Xinyou Done, thanks.

**Comment by fidencio** [comment]
@ZhangShuaiyi, may I ask you to rebase this PR atop of man? It'll make the CI happier.

**Comment by ZhangShuaiyi** [comment]
Hi @fidencio, I have ran "git rebase -i main" in my pr branch.  Do my other prs need rebase too?

**Comment by Ji-Xinyou** [comment]
I close and reopen to rerun the gha and I saw that the failing CI on aks is commonly seen on multiple PRs.

**Comment by ZhangShuaiyi** [comment]
Hi @Ji-Xinyou  @studychao  @fidencio , please take a look at this. Thanks!

**Pull request closed without merging**